### ISSUE TITLE
Dialogs with ImageListView on High-DPI displays switch to tiny pixels

### ISSUE THREAD

**Issue by UweKeim** [oozcitak/imagelistview#188]
Having a WinForms application that is _not_ [DPI aware](https://msdn.microsoft.com/de-de/library/windows/desktop/dn469266(v=vs.85).aspx) (i.e. the manifest contains [`<dpiAware>False</dpiAware>`](https://stackoverflow.com/questions/13448180/how-do-i-enable-dpiaware)). the application runs well and correctly auto-scaled by Windows 10.

When opening a dialog that contains an ImageListView instance, suddenly the application's automatic scale is switched to 100% making everything look very tiny.

This only happens with ImageListView. All other dialogs with other controls, even third-party and even a hosted Windows File Explorer control does _not_ trigger the switching of the DPI aware mode.

**My question:**

Is there a way to stop ImageListView from altering the DPI awareness of my application?

**Edit 1:**

The same happens with the "ImageListViewDemo" application as soon as selecting a folder that contains images.

**Edit 2:**

Here is a side-by-side comparison image:

![image](https://cloud.githubusercontent.com/assets/415439/22115354/ea2bd0ea-de6c-11e6-9fb9-d46b71ba0e2f.png)

**Edit 3:**

I've hat suspision that the WIC (Windows Imaging) may cause this and set 

    UseWIC = UseWIC.Never;

but unfortunately this did not change anything.

**Edit 4:**

When compiling the "ImageListViewDemo" with the build configuration "**Release without WIC**", the error does _not_ appear.

**Comment by UweKeim** [comment]
My currently working workaround is to _not_ use your NuGet package.

Instead, I'm compiling my own version based on your master branch with the "Release without WIC" build configuration.

**Comment by oozcitak** [comment]
Hi Uwe,

The problem is indeed with WPF/WIC. It appears that it is not a good idea mixing WPF and winforms. Disabling WIC solves the issue.

But it is also possible to still use WIC but declare your app DPI aware. This actually results in the control scaling nicely instead of the ugly bitmap up-scaling of the OS.

Here is the modified main function that fixes the demo project in the repository:

``` csharp
    class Program
    {
        [STAThread]
        static void Main(string[] args)
        {
            if (Environment.OSVersion.Version.Major >= 6) SetProcessDPIAware();
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            Application.Run(new DemoForm());
        }

        [System.Runtime.InteropServices.DllImport("user32.dll")]
        private static extern bool SetProcessDPIAware();
    }
```
It should also be possible to achieve the same with a manifest. IDK if you can/want to do this with your app though.

**Comment by oozcitak** [comment]
I don't have an immediate fix for this issue. I will include the "Release without WIC" build in the NuGet package though.

**Comment by UweKeim** [comment]
Thanks a lot, Ozgur!

**Comment by oozcitak** [comment]
I believe I fixed this issue with the latest 11.1.0 release. The `UseWIC` property of the control now causes the WPF library to be loaded dynamically. If you set `UseWIC = false` while initializing the form, WPF functions shouldn't be loaded at all and the DPI awareness of the application should remain unchanged. It is important to not change the `UseWIC` property while application is running.

The `USEWIC` conditional compilation symbol is also still available and it conditionally disables all WPF/WIC functionality as before.
